dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -0,0 +1,756 @@
+package io.snappydata.hydra.cluster;
+
+
+import com.gemstone.gemfire.LogWriter;
+import com.gemstone.gemfire.SystemFailure;
+import hydra.*;
+import org.apache.spark.SparkContext;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.sql.SnappyContext;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.rmi.RemoteException;
+import java.util.*;
+import java.io.*;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import scala.Predef;
+import scala.Tuple2;
+import scala.collection.JavaConverters;
+import scala.collection.immutable.Map;
+
+import util.TestException;
+
+import static java.lang.Thread.sleep;
+
+public class SnappyTest implements Serializable {
+
+    private static transient JavaSparkContext jsc;
+    private static transient SnappyContext snc;
+    protected static SnappyTest snappyTest;
+    private static char sep;
+    private static HostDescription hd;
+    private static String snappyPath = null;
+    private static String logFile = null;
+    private static String localHost = null;
```

Comment:
no need to declare it here .instead  use  HostHelper.getLocalHost() whenever required